src/compiler/abepi.rs

```diff
@@ -262,11 +278,37 @@ impl<F: From<u64> + Into<u32> + Clone, V: Clone> CompilationUnit<F, V> {
 
     fn compile_expression_or(
         &self,
-        _dsym: DebugSymRef,
-        _lhs: Expression<F, V>,
-        _rhs: Expression<F, V>,
+        dsym: DebugSymRef,
+        lhs: Expression<F, V>,
+        rhs: Expression<F, V>,
     ) -> CompilationResult<F, V> {
-        todo!()
+        let mut sub = Vec::new();
+
+        flatten_bin_op(BinaryOperator::Or, lhs, rhs, &mut sub);
+        assert!(sub.iter().all(|se| se.is_logic()));
+
+        let sub = sub
+            .iter()
+            .map(|se| self.compile_expression_logic(se.clone()))
+            .collect::<Vec<_>>();
+
+        // By De Morgan's law, a or b = not (not a and not b)
+        // In OneZero 0F 1T
+        // If !a, !b are 1T => !a * !b = 1T, if any of !a, !b is 0F => !a * !b = 0F => (1-a) * (1-b)
+        // = 0F So we can do the AND of the negated expressions
+        // And then negate the result
+        let one_zero = sub
+            .iter()
+            .skip(1)
+            .fold(sub[0].one_zero.clone().one_minus(), |acc, se| {
+                acc * se.one_zero.clone().one_minus()
+            });
+
+        CompilationResult {
+            dsym,
+            anti_booly: one_zero.clone(),
```

Comment:
@rutefig perhaps there is an simpler way of doing the anti_booly version based on anti_booly values.

Just multiplication, if one is 0T, the result will be 0T. If all are 0>T, the result will be 0>T. Because we re in a prime field multiplications of 0> will always be 0>.

Reply:
We will leave it for later - compare approaches to check which one is more performant